test/lib/rails-jquery-autocomplete/autocomplete_test.rb

```diff
@@ -77,6 +79,7 @@ class AutocompleteTest < Test::Unit::TestCase
         should 'add that extra data to result' do
           item = mock(Object)
           mock(item).send(:name).times(2) { 'Object Name' }
+          stub(item).kind_of?(Hash){ false }
```

Comment:
Prefer `Object#is_a?` over `Object#kind_of?`.<br>Space missing to the left of {.